code/modules/core_implant/cruciform/machinery/wall_obelisk.dm

```diff
@@ -17,7 +17,7 @@
 	var/active = TRUE //Were always active
 
 	var/area_radius = 7
-	var/damage = 120
+	var/damage = 40 //120 damage is insane, but these are a rarer resource (for now at least). Let's try 40
```

Comment:
Im greatly against this nerf.
these are located in areas like dorms, cryo spawn, and other RP related areas that should get some protection.

Reply:
The problem with that is that they also protect gamer areas like the scrap beacon. What if I tripled their starting ammo to make up for the thirding of their damage? Then I could in a separate PR later make another subtype that's got less ammo for the places where they're providing mechanical advantage, making those areas more reliant on Church upkeep to remain roachproof. (I don't want to touch mapping in this PR.)

Reply:
It's also worth noting that 40 is still enough to one-shot gib most roaches, it's only spiders and termites that this has a significant effect.

Reply:
Scrap beacon is a none factor, 40 and 120 both kill the roaches that are those lockers. its affects have no difference.
The only difference this change makes is affecting areas that *should be protected* less so, rather then gamer areas.